inspire_relations/command_producers.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+def produce_labels_string(labels):
+    if labels:
+        return ''.join([':' + label for label in labels if label])
+    else:
+        return ''
```

Comment:
There's no need to do something special for the empty list:
```python
In [1]: ''.join([])
Out[1]: ''
```
If you are guarding against `None` there, or another falsy value different from empty list, then you should amend your callers.